### PR TITLE
Dictionary 1.13 release notes

### PR DESCRIPTION
**Pull request by hknahal**

### CODE REVIEW

File: docs/release-notes/dictionary-releases.md [hunk 1]
```diff
@@ -20,6 +20,31 @@ None to report.
...
 
+## Release 1.13
+
+**Release Date: November 17, 2021**
+
+### Updates
+
+- The "AJCC 6th Edition" has been added to the controlled terminology for tumour staging systems. This will allow data submitters to submit TNM categories and stage groups for the AJCC 6th Edition.
+- Optional clinical fields related to dose intensity reduction have been added to the Chemotherapy table. These include `dose_intensity_reduction`, `dose_intensity_reduction_event` and `dose_intensity_reduction_amount`.
+- Optional clinical fields related to laterality have been added. These include the `laterality` field in the Primary Diagnosis table and the `specimen_laterality` field in the Specimen table.
```
Comment (kcullion) [inline docs/release-notes/dictionary-releases.md:31]: should we say "file" instead of "table"? does the audience know this as a "table"?

**Comment by hknahal** [comment]
@joneubank I just noticed in the Dictionary Preview that the new Chemotherapy fields show up, but the other original ones don't show anymore. Any idea what might be causing this? The JSON dictionary looks okay to me and didn't throw any errors when I compiled it or added it to Lecturn:
missing_chemotherapy_fields

**Review by kcullion — DISMISSED** [review]

**Review by joneubank — APPROVED** [review]

File: docs/release-notes/dictionary-releases.md (hunk 1 above)
Comment (hknahal) [reply docs/release-notes/dictionary-releases.md:31]: @kcullion Rosi was using "table" in the past Release notes, so I thought maybe to stay consistent to keep using "table".

Comment (kcullion) [reply docs/release-notes/dictionary-releases.md:31]: Ok let's keep it as table then! thanks :)

**Review by ciaranschutte — APPROVED** [review]

**Pull request merged**